src/init.cpp

```diff
@@ -1535,9 +1531,9 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     uiInterface.InitMessage(_("Done loading"));
 
 #ifdef ENABLE_WALLET
-    if (pwalletMain) {
+    if (CWallets::defaultWallet()) {
         // Run a thread to flush wallet periodically
-        threadGroup.create_thread(boost::bind(&ThreadFlushWalletDB, boost::ref(pwalletMain->strWalletFile)));
+        threadGroup.create_thread(boost::bind(&ThreadFlushWalletDB, boost::ref(CWallets::defaultWallet()->strWalletFile)));
```

Comment:
Jup, @luke-jr did this. (Create a loop in `ThreadFlushWalletDB` over all wallets)